drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -1382,7 +1382,7 @@ void RasterizerCanvasGLES3::_render_batch(Light *p_lights, uint32_t p_index) {
 				GLuint vertex_array_gl = 0;
 				GLuint index_array_gl = 0;
 
-				uint32_t input_mask = 0; // 2D meshes always use the same vertex format
+				uint64_t input_mask = 0; // 2D meshes always use the same vertex format
```

Comment:
While changing this line:
```suggestion
				uint64_t input_mask = 0; // 2D meshes always use the same vertex format.
```